src/server/graphql/resolvers/index.js

```diff
@@ -127,6 +128,10 @@ export async function resolveProjectUserSummaries(projectSummary, args, {rootVal
   })
 }
 
+export function resolveStartOfWeek(cycle) {
+  return moment(cycle.startTimestamp).startOf('isoweek').toDate()
```

Comment:
Like the generic naming...let's call `cycle` `parent` instead to suggest that it could be reused with any model having a starting timestamp. Ex:
```js
export function resolveStartOfWeek(parent) {
  return parent.weekStartedAt || moment(parent.startedTimestamp).startOf('isoweek').toDate()
}
```